octofree/main.py

```
@@ -165,17 +291,34 @@ def main():
             except Exception as e:
-                logging.warning(f"[SCRAPER_X] Error fetching from X.com (non-critical): {e}")
-                logging.info("[SCRAPER_X] Continuing with website data only")
+                logging.warning(f"🔴 [X SCRAPER] Error fetching from X.com (non-critical): {e}")
+                logging.info("  └─ Continuing with website data only")
         elif x_enabled:
-            logging.debug(f"[SCRAPER_X] Outside X.com check window (current hour: {datetime.now().hour}). Skipping X.com check.")
+            logging.debug(f"🔍 [SCRAPER_X] Outside X.com check window (current hour: {datetime.now().hour}). Skipping X.com check.")
         
         # Log summary of what was found
-        logging.info(f"SUMMARY: Total {len(current_sessions)} unique session(s) to process (Website: {len(website_sessions)}, X.com: {len(x_sessions)}, New from X.com: {len(new_x_sessions)})")
+        logging.info(f"📊 SUMMARY")
```

Ruff (0.14.1)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)